code/.deps/problems.md

```diff
@@ -0,0 +1,33 @@
+# Dependency analysis
+
+## UNRESOLVED Production dependencies
+
+1. `@vscode/deviceid@0.1.1`
+2. `@vscode/policy-watcher@1.1.8`
+3. `@vscode/proxy-agent@0.28.0`
+4. `@vscode/spdlog@0.15.1`
+5. `@vscode/tree-sitter-wasm@0.0.5`
+6. `@vscode/vscode-languagedetection@1.0.21`
+7. `@vscode/windows-process-tree@0.6.0`
+8. `@vscode/windows-registry@1.1.0`
+9. `@xterm/addon-clipboard@0.2.0-beta.79`
+10. `@xterm/addon-image@0.9.0-beta.96`
+11. `@xterm/addon-ligatures@0.10.0-beta.96`
+12. `@xterm/addon-progress@0.2.0-beta.2`
+13. `@xterm/addon-search@0.16.0-beta.96`
+14. `@xterm/addon-serialize@0.14.0-beta.96`
+15. `@xterm/addon-unicode11@0.9.0-beta.96`
+16. `@xterm/addon-webgl@0.19.0-beta.96`
+17. `@xterm/headless@5.6.0-beta.96`
+18. `@xterm/xterm@5.6.0-beta.96`
+
+## UNRESOLVED Development dependencies
+
+1. `@csstools/selector-specificity@3.0.1`
+2. `@ts-morph/common@0.20.0`
+3. `@vscode/telemetry-extractor@1.10.2`
+4. `@vscode/test-cli@0.0.6`
+5. `@vscode/test-web@0.0.62`
+6. `@vscode/vscode-perf@0.0.19`
+7. `innosetup@6.0.5`
+8. `postcss-nesting@12.0.2`
```

Comment:
Could you clarify what are the next steps for the `Unresolved` dependencies?

Reply:
@RomanNikitenko The file `code/.deps/problems.md` just for information. It shows current state. To remove libraries from unresolved dependencies list, we can harvest them using this link [harvest](https://clearlydefined.io/harvest) or add libraries as exclusions in `code/.deps/EXCLUDED/prod.md` or `code/.deps/EXCLUDED/dev.md` files.

For example: https://github.com/eclipse-che/che-dashboard/blob/main/.deps/EXCLUDED/dev.md

Reply:
There are 12 unresolved deps with `@vscode` prefix - could you clarify **_why_** they are unresolved?
did you try to apply `harvest` approach for them?

Reply:
I have fixed it